File: programmers/js/basic/181879.js
// 길이에 따른 연산
function solution(num_list) {
  if (num_list.length >= 11) {
    return num_list.reduce((acc, cur) => acc + cur, 0);
  } else {
    return num_list.reduce((acc, cur) => acc * cur, 1);
  }
}

File: programmers/java/basic/181879.java
// 길이에 따른 연산

import java.util.Arrays;

class Solution {
    public int solution(int[] num_list) {
        if (num_list.length > 10) {
            return Arrays.stream(num_list).sum();
        } else {
            return Arrays.stream(num_list)
                .reduce(1, (acc, cur) -> acc * cur);
        }
    }
}